src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -426,37 +426,14 @@ private bool IsSupported(WindowTransparencyLevel level, Version windowsVersion)
 
         private void SetTransparencyTransparent(Version windowsVersion)
         {
-            // Transparent only supported with composition on Windows 8+.
-            if (!_isUsingComposition || windowsVersion < PlatformConstants.Windows8)
+            // Transparent only supported with composition on Windows 10+.
+            if (!_isUsingComposition || windowsVersion < PlatformConstants.Windows10)
                 return;
 
-            if (windowsVersion < PlatformConstants.Windows10)
```

Comment:
@grokys

Reply:
@trympet I haven't double-checked this part, but it might be a problem to drop transparency on pre-win10.0.15063 without SetWindowCompositionAttribute without implementing Direct Composition now. As it's way more important that blur itself.

Reply:
Interesting. Sounds like the biggest hurdle is getting the backdrop from DWM.

If you want, I can revert the commit where the blur was altered and link to `SetWindowCompositionAttribute` dynamically with GetProcAddress instead 😄

Reply:
Resolved in 544cf0f + 8f7cf7c